app/views/devise/registrations/new.html.erb

```diff
@@ -1,34 +1,34 @@
 <h2>Sign up</h2>
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: {turbo: false }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+<%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
- [ ] Kindly fix the duplicated views' code that causes this UI issue.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/219825427-2f671fdd-193f-41b2-a641-9fadb86c50d0.png)